.github/workflows/semantic-release.yaml

```diff
@@ -1,39 +1,67 @@
-name: Semantic Release
-
-on:
-  push:
-    branches:    
-      - master
-
-jobs:
-  release:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        persist-credentials: true
-
-    - uses: actions/setup-node@v2
-      with:
-        node-version: '14'
-
-    - run: yarn install --frozen-lockfile
-
-    - name: Import GPG key
-      uses: crazy-max/ghaction-import-gpg@v3.0.1
-      with:
-        gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-        passphrase: ${{ secrets.GPG_PASSPHRASE }}
-        git-user-signingkey: true
-        git-commit-gpgsign: true
-
-    - name: Build
-      run: yarn run build
-
-    - run: npx semantic-release
-      env:
-        GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        GIT_COMMITTER_NAME: github-actions[bot]
-        GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
\ No newline at end of file
+name: Semantic Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Build and release library
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - run: yarn install --frozen-lockfile
+
+      - name: Build
+        run: yarn run build
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v3.0.1
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+          git-user-signingkey: true
+          git-commit-gpgsign: true
+
+      - run: npx semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GIT_COMMITTER_NAME: github-actions[bot]
+          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+
+
+  update-tokens:
+    name: Update public JSONBin
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2

```

Comment:
Can you add a comment here that mentions that this task doesn't have access to the `dist` folder. This runs in parallel with the other job and the checkout step checks out the original HEAD (so before the tag). 
Just to avoid confusion for other devs in future :)

Reply:
Oh, I haven't thought about parallel running. Sure, I'll add the comment.